index.html

```diff
@@ -0,0 +1,201 @@
+      <td>authors</td>
+      <td id=spec-md-authors>
+        <ul class=spec-authors-list>
+          <li><a rel=author href=mailto:ashi@apollographql.com>Ashi Krishnan</a>
+        </ul>
+      </td>
+    </tr>
+  </table>
+</header>
+
+## Problem
+
+Some specifications—such as the [federation specification](../federation-spec) and [CSDL specification](./csdl)—apply to GraphQL schema documents. They may place additional requirements on conforming documents. Such specifications may also introduce new types, directives, and other schema objects which can be used within those documents.
+
+The [GraphQL Schema Definition Language](https://spec.graphql.org/) does not provide a way to declare that a schema document implements additional specs beyond the GraphQL SDL. Currently, this information is derived from context, or through some explicit, out-of-document paramenter provided to a spec processor. As a consequence, given only the text of an SDL document, there is no reliable way to discern what specifications—and what version of those specifications—the document conforms to.
+
+This specification provides a solution in the form of a single new well-known directive—the `@using` directive. This directive allows spec authors, schema authors, and schema processors to come to an agreement on which specifications a schema document is using.
+
+## Actors
+
+```mermaid
+#::[Actors who may be interested in the `@using` directive]
+graph TB
+  author("👩🏽‍💻 🤖  &nbsp;Author")-->schema(["📄  Schema"])
+  schema-->proc("🤖 &nbsp;Processor")
+  schema-->reader("👩🏽‍💻  Reader")
+  proc-->output(["📄  Output Schema"])
+```
+
+- **Authors (either human or machine)** write one or more `@using` directives into schema files to declare that the schema conforms to some set of specifications
+- **Machine processors** read the `@using` directives and compare them to a list of supported schema specs, activating the spec-mandated appropriate behaviors
+- **Human readers** see the `@using` directives and can follow the URLs to the specification, receiving an explanation of the requirements of the specification and what new directives, types, and other schema objects are available within the document.
+
+## The `@using` directive
+
+### `@using` within a document
+
+```graphql
+#::[Declaration of the `@using` directive]
+directive @using(spec: using_URL!)
+  repeatable on SCHEMA
+```
+
+The `@using` directive is applied to `schema` declarations to declare that the schema uses a particular spec. For example,
+
+```graphql
+#::[Using the `@using` directive]
+schema @using(spec: "https://spec.example.com/exampleSpec/1") {
+  query: Query
+}
+```
+
+### Requiring `@using` within a spec
+
+Specifications **may** include the following language to instruct schema authors to use `@using` to indicate documents which are intended to conform to the specification:
+
+> SPEC_NAME schema documents **must**/**should**/**may** have a `schema` declaration with a [`@using` directive](/specs/use)
```

Comment:
Link needs to be to `using` (same in next blockquote)

Reply:
fixed